extension/lib/bigcommerce/customer/shapeRequestError.js

```diff
@@ -0,0 +1,28 @@
+const BigCommerceRequestClientError = require('./request/ClientError')
+const BigCommerceRequestUnparseableMessageError = require('./request/UnparsableMessageError')
+
+module.exports = function shapeRequestError (err) {
+  // Try parsing the (potential) underlying api error
+  const errorMessageMatch = err.message.match(/({.+})/)
+  if (!errorMessageMatch) {
+    return err
+  }
+
+  let parsed
+  try {
+    parsed = JSON.parse(errorMessageMatch[1])
```

Comment:
... but then you have to use the 0th index here.
```suggestion
    parsed = JSON.parse(errorMessageMatch[0])
```